todo/views/team.py

```
@@ -0,0 +1,85 @@
+            fallback_response = ApiErrorResponse(
+                statusCode=500,
+                message=ApiErrors.UNEXPECTED_ERROR_OCCURRED,
+                errors=[{"detail": str(e) if settings.DEBUG else ApiErrors.INTERNAL_SERVER_ERROR}],
+            )
+            return Response(
+                data=fallback_response.model_dump(mode="json"), 
+                status=status.HTTP_500_INTERNAL_SERVER_ERROR
+            )
+
+    def _handle_validation_errors(self, errors):
+        from todo.dto.responses.error_response import ApiErrorDetail, ApiErrorSource
```

Pylint (3.3.7)

[error] Unable to import 'todo.dto.responses.error_response'

(E0401)